documentation/docs/pages/developers/nymvpncli.mdx

```diff
@@ -0,0 +1,278 @@
+import { Callout } from 'nextra/components'
+
+# Nym VPN CLI
+
+This is a short guide to setting up and using the `nym-vpnc` tool, which is used in conjunction with the `nym-vpnd` daemon.
+
+<Callout type="warning" emoji="⚠️">
+  These binaries have superceded the older `nym-vpn-cli` binary. This still operates for the moment as it is being used in testing scenarios but will go out of date quickly.
+</Callout>
+
+Download and run instructions for the GUIs can be found [here](https://nymvpn.com/en/download/linux).
+
+## Download & Extract Binary
+Check the [release page](https://github.com/nymtech/nym-vpn-client/releases/) page for the latest release version and modify the instructions accordingly. These instructions use the latest as of the time of writing.
+```sh
+wget -q https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-core-v1.1.0-beta.3/nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
+tar -xzf nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
```

Comment:
or alternatively make it a bit more eye banging that a user has to think and replace:

`tar -xzf <BINARY>.tar.gz`